server/src/internalClusterTest/java/org/elasticsearch/snapshots/GetSnapshotsIT.java

```diff
@@ -244,9 +306,17 @@ private static void assertStablePagination(String repoName, Collection<String> a
         Collection<String> allSnapshotNames,
         String repoName,
         GetSnapshotsRequest.SortBy sortBy,
-        SortOrder order
+        SortOrder order,
+        String... extraPatterns
```

Comment:
I think this is the name patterns, could we call it `namePatterns`?

Reply:
Renamed :)